src/binary/ibc-relayer/commands/start.ts

```diff
@@ -14,6 +17,55 @@ import { resolveKeyFileOption } from '../../utils/options/shared/resolve-key-fil
 import { resolveMnemonicOption } from '../../utils/options/shared/resolve-mnemonic-option';
 import { signingClient } from '../../utils/signing-client';
 
+function resolveHeights(
+  {
+    scanFromSrc,
+    scanFromDest,
+    home,
+  }: {
+    scanFromSrc: number | null;
```

Comment:
Can we make this struct some other type, like `LoggerFlags` or `LoopFlags` are?

Reply:
2fa51eb

I extracted it to params type so it's easier to read, however, this type is not reused anywhere.